Automattic/wp-calypso
# Stats: Add Tracks event for "views" of the inline feedback card

Related to: https://github.com/Automattic/red-team/issues/265#issuecomment-2533524014

## Proposed Changes

Adds a new Tracks event for "views" of the inline feedback card. In this context, it means when the component first appears in the browser view as opposed to just the initial render. The new Tracks event is:

- `stats_feedback_action_view_persistent_section`

## Why are these changes being made?


Better measuring!

## Testing Instructions



* Check out the branch.
* Add a log statement around line 183.
* Confirm the log statement is printed when the view is first visible on screen.
* Confirm it does not fire multiple times for a give page view. (ie: scrolling the component in and out of view)

## Pre-merge Checklist
